### ISSUE TITLE
dependencies not met

### ISSUE THREAD

**Issue by NovaGL** [ebaauw/homebridge-zp#11]
Just a quick note to say when I try and run this it doesn't work.

This is because you don't have request and xmljs in the package.json

```
  "dependencies": {
    "sonos": "^0.12.1",
    "request": "^2.65.0",
    "xml2js": "0.4.17"
  },
```

This means when doing npm install now everything should work fine.

**Comment by ebaauw** [comment]
That's odd, I install homebridge-zp to my server using npm and never ran into this.  I'm still figuring out npm, thought.  The sonos module uses request and xml2js, so they are installed.  What npm version are you using?

Anyway, @NovaGL you're correct: I should mention my direct dependencies in package.json.

**Comment by ebaauw** [comment]
Fixed in v0.1.2.
